test/hw3/test_task4.py

```diff
@@ -0,0 +1,19 @@
+from All_home_works.hw3.task4 import is_armstrong
+
+
+def test_is_armstrong_case1():
+    assert is_armstrong(153)
+
+
+def test_is_armstrong_case2():
+    assert not is_armstrong(10)
+
+
+def test_is_armstrong_case3():
+    assert is_armstrong(9)
+
+
+if __name__ == "__main__":
```

Comment:
Why did you need an additional test runner?

Reply:
This case is in an Example: "
Examples:
- 9 is an Armstrong number, 9 = 9^1 = 9
- 10 is not: 10 != 1^2 + 0^2 = 1
- 153 is : 153 = 1^3 + 5^3 + 3^3 = 1 + 125 + 27 = 153
".
So I decided to check it.

Reply:
`if __name__ == "__main__":`  is for executable files, you have have test runner you don't need it.  

Please make sure that there is not such line in the tests of the other tasks.